content/hardware/03.nano/boards/nano-matter/tutorials/08.ml-magic-wand/content.md

```diff
@@ -0,0 +1,663 @@
+tags:
+  - AI
+  - ML
+  - IMU
+  - Modulino
+  - Matter
+
+author: 'Christopher Méndez'
+hardware:
+  - hardware/03.nano/boards/nano-matter
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+---
+
+## Overview
+
+This tutorial describes how to build a gesture recognition system based on a machine learning model using TensorFlow and the Arduino Nano Matter.
+
+![Nano Matter Magic Wand](assets/thumbnail.png)
+
+The Arduino Nano Matter acts as a digital magic wand 🪄, where sensor data from its movements is processed by a model to classify and detect specific gestures. The inference results will turn on a LEDs array on a specific color respectively to the detected gesture.
+
+### Goals
+
+The goal of this project tutorial is to showcase the capabilities of the Arduino Nano Matter running Tiny Machine Learning models on the edge for gesture recognition. 
+The wand can detect two gestures drawn in the air: **"W" (wing gesture)** and **"O" (ring gesture)**.
+
+![Project working demo](assets/ani2.gif)
+
+## Hardware and Software Requirements
+
+![Hardware Required](assets/hardware.png)
+
+### Hardware Requirements
+
+- [Arduino Nano Matter](https://store.arduino.cc/products/nano-matter) (x1)
+- Modulino Movement (x1)
+- Modulino Pixels (x1)
+- Qwiic cables (x2)
+- [USB-C® cable](https://store.arduino.cc/products/usb-cable2in1-type-c) (x1)
+- [Custom 3D printed parts](assets/3d-files.zip)
+
+### Software Requirements
+
+- [Arduino IDE 2.0+](https://www.arduino.cc/en/software) or [Arduino Cloud Editor](https://create.arduino.cc/editor)
+- [Modulino library](https://github.com/arduino-libraries/Modulino). This library adds the support for the Modulinos, you can install it from the **Library Manager** in the Arduino IDE.
+- [Silicon Labs core](https://github.com/SiliconLabs/arduino). This enables the Silicon Labs hardware including the Arduino Nano Matter support. You can install it from the **Boards Manager** in the Arduino IDE. 
+
+### Download the Project Code
+
+[![ ](assets/download.png)](assets/magic_wand_modulino.zip)
+
+Download the complete project code [here](assets/magic_wand_modulino.zip).
+
+### Board Core and Libraries
+
+The **Silicon Labs** core contains the libraries and examples you need to work with the board's components, such as its Matter, Bluetooth® Low Energy, and I/Os. To install the Nano Matter core, navigate to **Tools > Board > Boards Manager** or click the Boards Manager icon in the left tab of the IDE. In the Boards Manager tab, search for `Nano Matter` and install the latest `Silicon Labs` core version.
+
+![Installing the Silicon Labs core in the Arduino IDE](assets/bsp-install.png)
+
+***The core version for this tutorial must be 2.3.0 or greater for the support of the Silabs Tensorflow Lite library.***
+
+## How to Magic Wand Works?
+
+In its **idle state**, the wand's LEDs remain **solid blue**, indicating that it is ready to detect a gesture.
```

Comment:
I would suggest adding a picture showing the wand's LED colors for the user's reference.